### ISSUE TITLE
Toolbar error when using dev-master

### ISSUE THREAD

**Issue by ShinDarth** [symfony/symfony#20829]
The dev toolbar is broken in dev-master, it gives error:

``` Block "menu" on template "@WebProfiler/Profiler/base.html.twig" does not exist in @WebProfiler/Profiler/base.html.twig at line 17.```

that happens only  when using:

``"symfony/symfony": "dev-master",``

problem is not there using:

``"symfony/symfony": "~3.2.0",``

**Comment by stof** [comment]
@javiereguiluz I suspect this is rleated to using Twig 2.x, which does not allow rendering non-existent blocks anymore.

**Comment by ShinDarth** [comment]
thanks for the PR, but unfortunately the problem persists

**Comment by ShinDarth** [comment]
Thanks for the PR, unfortunately the bug persists.

```
"name": "symfony/symfony",
            "version": "dev-master",
            "source": {
                "type": "git",
                "url": "https://github.com/symfony/symfony.git",
                "reference": "9091a82b66c6cf70d672868bedaf0a92be884c51"
            },
```

```
name": "twig/twig",
            "version": "2.x-dev",
            "source": {
                "type": "git",
                "url": "https://github.com/twigphp/Twig.git",
                "reference": "31801a18cee4ae16e013a7e1623ce999c96b3fa3"
            },
```

**Comment by stof** [comment]
@ShinDarth the fix is in 2.7, 2.8, 3.1 and 3.2 branches, but not yet in master. Test it with the Symfony 3.2 branch

**Comment by ShinDarth** [comment]
@stof as I said in the report, there is no problem using 3.2 (even before the PR)
